simapp/v2/simdv2/cmd/commands.go

```diff
@@ -76,7 +76,7 @@ func initRootCmd(
 		// snapshot.Cmd(newApp),
 	)
 
-	server.AddCommands(rootCmd, log.NewNopLogger(), tempDir(), nil) // TODO: How to cast from AppModule to ServerModule 
+	// server.AddCommands(rootCmd, log.NewNopLogger(), tempDir(), nil) // TODO: How to cast from AppModule to ServerModule 
```

Comment:
this isn't a solution, there are no server commands added to the root cmd here.

Reply:
readd serverv2